bundles/org.openhab.binding.resol/src/main/resources/ESH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="resol"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<bridge-type id="vbuslan">
+		<label>Bridge VBusLAN adapter</label>
+		<description>This bridge represents the Resol VBus-LAN adapter
+		</description>
+
+		<config-description>
+			<parameter name="ipAddress" type="text" required="true">
+				<context>network-address</context>
+				<label>IP Address</label>
+				<description>The IP address of the of the VBus-LAN adapter.</description>
+			</parameter>
+			<parameter name="port" type="integer" required="false" min="1024" max="49151">
```

Comment:
Where does the max value come from?

Reply:
very good question. I guess it is copy-paste from one of the other bindings I have checked. Maybe org.openhab.binding.vitotronic - which uses the same value. I'll replace it by 65535.